ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/board.c

```diff
@@ -0,0 +1,139 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2020 Scott Shawcroft for Adafruit Industries
+//
+// SPDX-License-Identifier: MIT
+
+#include "supervisor/board.h"
+#include "mpconfigboard.h"
+#include "shared-bindings/board/__init__.h"
+#include "shared-bindings/busio/I2C.h"
+#include "shared-bindings/dotclockframebuffer/DotClockFramebuffer.h"
+#include "shared-bindings/dotclockframebuffer/__init__.h"
+#include "shared-bindings/framebufferio/FramebufferDisplay.h"
+#include "shared-bindings/microcontroller/Pin.h"
+#include "shared-module/displayio/__init__.h"
+#include "boards/espressif_esp32s3_lcd_ev_v1.5/board.h"
+
+#define MP_DEFINE_BYTES_OBJ(obj_name, bin) mp_obj_str_t obj_name = {{&mp_type_bytes}, 0, sizeof(bin) - 1, (const byte *)bin}
```

Comment:
This is breaking the CI. Have you built and tested this locally?

Reply:
Not yet. 
I update  macro name as commit 4a5624a by @dhalbert.